### ISSUE TITLE
False positives for App Services site config templates

### ISSUE THREAD

**Issue by BernieWhite** [microsoft/PSRule.Rules.Azure#533]
**Description of the issue**

When using templates a number of forms can be used to set site config. When templates used the `Properties.siteConfig` property instead of a nested sub-resource the properties are ignored, failing a number of rules incorrectly.

Additionally, it is possible to use multiple site config sub-resources for other purposes such as site authentication. These should be ignored.

**Expected behaviour**

Update rules to use `Properties.siteConfig` settings when not configured by a web site config sub-resource.

**Module in use and version:**

- Module: PSRule.Rules.Azure
- Version: **v0.17.0-B2010006**

Captured output from `$PSVersionTable`:

```text
Name                           Value
----                           -----
PSVersion                      7.0.3
PSEdition                      Core
GitCommitId                    7.0.3
OS                             Microsoft Windows 10.0.19042
Platform                       Win32NT
PSCompatibleVersions           {1.0, 2.0, 3.0, 4.0…}
PSRemotingProtocolVersion      2.3
SerializationVersion           1.1.0.1
WSManStackVersion              3.0
```
